code/__DEFINES/xeno.dm

```diff
@@ -719,5 +719,5 @@
 
 #define FRENZY_DAMAGE_MULTIPLIER 2
 
-#define JOIN_AS_FACEHUGGER_DELAY (3 MINUTES)
-#define JOIN_AS_LESSER_DRONE_DELAY (30 SECONDS)
+#define JOIN_AS_FACEHUGGER_DELAY (1 MINUTES)
+#define JOIN_AS_LESSER_DRONE_DELAY (1 MINUTES)
```

Comment:
I am fine with an increased lesser drone delay.

I do not think we need to make face huggers spawn faster when they already can spawn instantly if they successfully hug.